src/renderer/src/store/reducers/SettingsReducer/index.ts

```diff
@@ -43,34 +72,7 @@ const settingsReducerMigrations = {
     },
     _persist,
   }),
-  1: (state: any) => ({
-    ...state,
-    data: {
-      ...state.data,
-      customNetworks: {
-        ...state.data.customNetworks,
-        polygon: [],
-      },
-      selectedNetworkByBlockchain: {
-        ...state.data.selectedNetworkByBlockchain,
-        polygon: DEFAULT_NETWORK_BY__BLOCKCHAIN.polygon,
-      },
-      networkProfiles: state.data.networkProfiles.map(profile => ({
-        ...profile,
-        networkByBlockchain: {
-          ...profile.networkByBlockchain,
-          polygon: DEFAULT_NETWORK_BY__BLOCKCHAIN.polygon,
-        },
-      })),
-      selectedNetworkProfile: {
-        ...state.data.selectedNetworkProfile,
-        networkByBlockchain: {
-          ...state.data.selectedNetworkProfile.networkByBlockchain,
-          polygon: DEFAULT_NETWORK_BY__BLOCKCHAIN.polygon,
-        },
-      },
-    },
-  }),
+  1: updatePolygonNetworks,
```

Comment:
Follow the same pattern. If you want to identify what this migration is doing, insert a comment above function